CriteoPublisherSdk/Sources/Network/CR_ApiQueryKeys.h

```diff
@@ -21,60 +21,64 @@
 
 @interface CR_ApiQueryKeys : NSObject
 
-@property(class, nonatomic, readonly) NSString *appId;
-@property(class, nonatomic, readonly) NSString *bidSlots;
-@property(class, nonatomic, readonly) NSString *bidSlotsCachedBidUsed;
-@property(class, nonatomic, readonly) NSString *bidSlotsImpressionId;
-@property(class, nonatomic, readonly) NSString *bidSlotsIsInterstitial;
-@property(class, nonatomic, readonly) NSString *bidSlotsIsNative;
-@property(class, nonatomic, readonly) NSString *bidSlotsIsRewarded;
-@property(class, nonatomic, readonly) NSString *bidSlotsPlacementId;
-@property(class, nonatomic, readonly) NSString *bidSlotsSizes;
-@property(class, nonatomic, readonly) NSString *bundleId;
-@property(class, nonatomic, readonly) NSString *cdbCallEndElapsed;
-@property(class, nonatomic, readonly) NSString *cdbCallStartElapsed;
-@property(class, nonatomic, readonly) NSString *cpId;
-@property(class, nonatomic, readonly) NSString *deviceModel;
-@property(class, nonatomic, readonly) NSString *deviceIdType;
-@property(class, nonatomic, readonly) NSString *deviceId;
-@property(class, nonatomic, readonly) NSString *deviceIdValue;
-@property(class, nonatomic, readonly) NSString *deviceOs;
-@property(class, nonatomic, readonly) NSString *eventType;
-@property(class, nonatomic, readonly) NSString *ext;
-@property(class, nonatomic, readonly) NSString *feedbackElapsed;
-@property(class, nonatomic, readonly) NSString *feedbacks;
-@property(class, nonatomic, readonly) NSString *skAdNetwork;
-@property(class, nonatomic, readonly) NSString *skAdNetworkVersion;
-@property(class, nonatomic, readonly) NSString *skAdNetworkIds;
+@property(class, nonatomic, readonly) NSString *_Nonnull appId;
```

Comment:
maybe using `NS_ASSUME_NONNULL_BEGIN/END` would be easier for later modifications ?